src/main/resources/default/templates/biz/tenants/user-accounts-list.html.pasta

```diff
@@ -12,7 +12,7 @@
                     headerLeftAlignmentClass="align-items-stretch text-break break-word"
                     link="@apply('%s/%s', cardBaseUrl , account.getIdAsString())">
                 <i:block name="header">
-                    <div class="pl-2 pr-1 align-self-center">
+                    <div class="pl-3 pr-1 align-self-center">
```

Comment:
Why are `pl` and `pr` different?

Reply:
Because the card body (the right section here) itself has a padding of `1.25rem` that has to be factored in.